apache/superset
# feat: Adds virtualization option to antd based Table component

### SUMMARY

This PR adds a `virtualization` prop to the recently added ant based Table component.  Virtualization allows the tables to be used in cases where there are hundreds or thousands of columns, or large dataset with many thousands of rows with no pagination enabled.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

1. Checkout PR and open console at superset/superset-frontend
2. run `npm ci`
3. run `npm run storybook` and view docs and test virtualized table at: http://localhost:6006/?path=/story/design-system-components-table-overview--page under the "Virtualization for Performance" section, or interact directly with virtualized table instance at: http://localhost:6006/?path=/story/design-system-components-table-examples--virtualized-performance
4. Ensure the other table example stories work correctly
5. run `npm run dev-server` and go to http://localhost:9000/dataset/add/?testing to ensure the table instance used in the create dataset workflow still behaves as expected following instructions on this prior PR: https://github.com/apache/superset/pull/21948

### ADDITIONAL INFORMATION
